src/XrdCns/XrdCnsConfig.cc

```diff
@@ -150,6 +150,7 @@ int XrdCnsConfig::Configure(int argc, char **argv, char *argt)
                     else NoGo = NAPath("'-a'", Spec.argval);
                  break;
        case 'B': Opts |= optNoCns;
+                 [[gnu::fallthrough]];
```

Comment:
Fallthroughs must be explicitly marked. Note, that `[[fallthrough]]` attribute was added to C++17 and GNU makes it available in C++11 and C++14 via `[[gnu::fallthrough]]`. Note that Clang had for years `[[gnu::fallthrough]]`. One can also use GNU style attribute. They also support simple C++ comments with specific patterns. Read more here: https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html (Search for `-Wimplicit-fallthrough`).